docs/articles/documentation/extending-testcafe/reporter-plugin/reporter-methods.md

```diff
@@ -125,7 +129,8 @@ The `testRunInfo` object provides detailed information about the test run. The o
 
 Property            | Type             | Description
 ------------------- | ---------------- | --------------------------------------------------------
-`errs`              | Array of Strings | An array of errors that occurred during the test run.
+`errs`              | Array of Objects | An array of errors that occurred during the test run. Use the [formatError](helpers.md#formaterror) helper to stringify objects in this array.
```

Comment:
to stringify objects? stringify?

Reply:
Yes, stringify.
 
https://en.wiktionary.org/wiki/stringify
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Of course, we can say 'convert to strings' instead.

*Use the [formatError](helpers.md#formaterror) helper to convert objects in this array to strings.*